Utils/Campaign builder/Unit1.pas

```diff
@@ -302,6 +312,12 @@ procedure TForm1.seMapCountChange(Sender: TObject);
 begin
   if fUpdating then Exit;
 
+  //Bug fixes flaws SpinEdit component which makes checking the entered number after the OnChange event
+  //If you remove the "if (seMapCount.Value > MAX_CAMP_MAPS)or(seMapCount.Value < seMapCount.MinValue) then" there is a problem shown here:
+  //http://gifok.net/images/2017/02/14/2017-02-14_16-01-27.gif

```

Comment:
As I said above, this comment should be in the pull request description not the code.

Here you should write a comment like "//SpinEdit can call OnChange with a value outside of the valid range, so validate it first"